docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -42,13 +42,13 @@ Currently, Slack and email notifications are supported.
 
 1. Sign in to your Dagster Cloud account.
 
-2. Click the **user menu (your icon) > Cloud Settings**.
+2. In the top navigation, click **Deployment**
 
 3. Click the **Alerts** tab.
 
 4. Click **+ Create alert policy**.
 
-5. From the **Alert Policy Type** drop-down, select **Run Alert** or **Schedule/Sensor Alert**.
+5. From the **Alert Policy Type** drop-down, select **Run alert**, **Schedule/Sensor alert**, or **Agent downtime alert**.
```

Comment:
Would you mind also changing `Alert Policy Type` to `Alert policy type` while you're here? :) Nit picky, but I just noticed this is how it's labeled in the UI.